action.yml

```diff
@@ -145,4 +145,57 @@ runs:
           --pr-title "$PR_TITLE" \
           --pr-body "$PR_BODY" \
           --jira-key "$JIRA_KEY"
+      shell: bash
+
+    - name: Create Jira task for newly opened issues
+      if: ${{ github.event_name == 'issues' && github.event.action == 'opened' && github.event.repository.full_name == 'osbuild/image-builder-frontend' }}
```

Comment:
Should we also hardcode the repository name?

Reply:
IMHO, no repository name should be checked here. If a repository enables the "on issue opened" trigger with this action, the intention is that the action will perform its intended function. If you really want to make this explicit, you should add a new option to the action `inputs` and act based on its value.

https://github.com/osbuild/pr-best-practices/blob/f90b2a49f99d4fbd5ec5426edafd188f90546414/action.yml#L4-L13